src/main/java/school/lemon/changerequest/java/introduction/hw1/Task5.java

```diff
@@ -3,10 +3,15 @@
 
 public class Task5 {
     public static double calculateA(double x, double y, double z) {
-        return 0;
+        double a;
+        a = (2 * Math.sin(x-(Math.PI)/6)*calculateB(z));
+        a = a / (0.5 +Math.pow(Math.sin(y),2));
+        return a;
     }
 
     private static double calculateB(double z) {
-        return 0;
+       return 1 +((z*z)/(3+(z*z)/5));
```

Comment:
When you see that smth is calculated more than once, in most cases it'll be better to create variable for such expression.